src/controllers/update/relevant.period.combined.statements.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { getApplicationData } from "../../utils/application.data";
+import { getRegistrationDate } from "../../utils/update/relevant.period";
+import { CombinedStatementPageKey } from "../../model/relevant.period.combined.statements.model";
+import { InputDate } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.RELEVANT_PERIOD_COMBINED_STATEMENTS_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(config.RELEVANT_PERIOD_COMBINED_STATEMENTS_PAGE, {
+      backLinkUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
+      templateName: config.RELEVANT_PERIOD_COMBINED_STATEMENTS_PAGE,
+      ...appData,
+      dateOfCreation: getRegistrationDate(appData.update?.date_of_creation as InputDate)
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.RELEVANT_PERIOD_COMBINED_STATEMENTS_PAGE}`);
+    const combinedStatementPageKey = req.body[CombinedStatementPageKey];
```

Comment:
Can we remove this line for now
`const combinedStatementPageKey = req.body[CombinedStatementPageKey];`
we need to think more about how to get the filled out check boxes from the request body. Probably as part of the next ticket

Reply:
@ConnorOS-ch ok sure will do.thank you